translate/storage/test_po.py

```diff
@@ -588,12 +587,11 @@ def test_multiline_obsolete(self):
     def test_merge_duplicates(self):
         """checks that merging duplicates works"""
         posource = '#: source1\nmsgid "test me"\nmsgstr ""\n\n#: source2\nmsgid "test me"\nmsgstr ""\n'
-        pofile = self.poparse(posource)
-        #assert len(pofile.units) == 2
+        pofile = self.poparse(posource, duplicatestyle="allow")
         pofile.removeduplicates("merge")
         assert len(pofile.units) == 1
         assert pofile.units[0].getlocations() == ["source1", "source2"]
-        print(pofile)
+
```

Comment:
Please remove this blank line. PEP8 recommends single blank line between methods.